### DISCUSSION TITLE
Webhooks --> gotify

### DISCUSSION THREAD

**Discussion by kimpenhaus** [alexjustesen/speedtest-tracker#1557] [Q&A] [answered]
I've setup to use a webhook to notify me via gotify:

<img width="1704" alt="grafik" src="https://github.com/alexjustesen/speedtest-tracker/assets/575109/2ad8e48f-1015-402e-b950-2f41b003d5cc">

the really stranghe thing is, that testing the webhook results in a notification:

<img width="1644" alt="grafik" src="https://github.com/alexjustesen/speedtest-tracker/assets/575109/7589eac5-ad0e-4e0a-b4e7-57840da30085">

but from the scheduled speetest no notification is arriving

don't know if that is expected as in #40 gotify is mentioned and not implemented yet. but I was wondering why the test is successful but not the notification during speedtests...

does anyone have any insights?

thanks,
marcus

**Comment by svenvg93** [comment]
Might be because the payload is diffrent for the test and completed/threshold notification. Gotify might not like the payload

**Comment by kimpenhaus** [comment]
ah - yeah ... you might be right

<img width="2676" alt="grafik" src="https://github.com/alexjustesen/speedtest-tracker/assets/575109/a5030a4c-bd9a-469c-9f99-001559f302dd">

gotify is not very helpful to trace down issues ...

thanks!

**Reply by svenvg93** [reply]
The notifications are not that hard to build, if you know the payload. you might be able to add support for it yourself :)

**Reply by svenvg93** [reply]
Was able to figure it out, will make a PR in the coming days for this 
<img width="741" alt="image" src="https://github.com/alexjustesen/speedtest-tracker/assets/4511676/45f484a9-ba9f-495b-891f-18cffca7ad8d">

**Reply by kimpenhaus** [reply]
oh that's nice - thank for the effort - love it 😄

**Reply by svenvg93** [reply, marked as answer]
PR is ready :) 
https://github.com/alexjustesen/speedtest-tracker/pull/1561
